.github/workflows/build_hub.yml

```diff
@@ -141,5 +155,43 @@ jobs:
         run: |
           false
 
+  build_docker:
+    name: Build docker image
+    needs:
+      - unit_tests
+      - linting
+      - docs
```

Comment:
I'm confused with the addition of all these lines in the job (158-195). These don't seem to have anything to do with linting?

Reply:
I also split some of the jobs out to allow them to run in parallel. This tells github actions to require these three jobs to pass before starting this one.

btw, is the reason for this job just to check that docker builds?

Reply:
Ah, I see it now.

Correct, this job verifies the Docker container builds along with all unit tests.